acceptance/openstack/networking/v2/extensions/layer3/layer3.go

```diff
@@ -246,3 +248,24 @@ func WaitForRouterInterfaceToDetach(client *gophercloud.ServiceClient, routerInt
 		return false, nil
 	})
 }
+
+func GetNetworkIdByName(client *gophercloud.ServiceClient, name string)(string,error){
```

Comment:
Isn't is possible to use the existing `IDFromName` function in the `networks` package?